aten/src/ATen/native/mkldnn/xpu/detail/Attr.h

```diff
@@ -460,4 +460,49 @@ static inline void construct_attr_by_post_op(
   }
 }
 
+enum class ScalingType : std::uint8_t {
+  TensorWise, // fp32 scales
+  RowWise, // fp32 scales
+  BlockWise1x16, // fp8_e4m3fn scales
+  BlockWise1x32, // fp8_e8m0fnu scales
+  BlockWise1x128, // fp32 scales
+  BlockWise128x128, // fp32 scales
+};
+
+static inline int get_onednn_mask_from_scaling_type(ScalingType scaling_type) {
+  // Set according to
+  // https://github.com/uxlfoundation/oneDNN/blob/main/tests/benchdnn/doc/knobs_attr.md#--attr-scales
+  switch (scaling_type) {
+    case ScalingType::TensorWise:
+      return 0;
+    case ScalingType::RowWise:
+      return 1 << 0;
+
+    // ChannelWise
```

Comment:
suggest to remove the unsupported ScalingType to keep the PR semantic is simple.